src/vue-components/inflections.vue

```diff
@@ -196,7 +196,7 @@
           this.selectedPartOfSpeech = newValue
           this.views = this.data.inflectionViewSet.getViews(this.selectedPartOfSpeech)
           this.selectedView = this.views[0]
-          if (!this.selectedView.hasPrerenderedTables) {
+          if (this.selectedView.isImplemented && !this.selectedView.hasPrerenderedTables) {
```

Comment:
This check duplicates several times - may be it is better to put it to outstanding computed property?